src/Examples.Console/Program.cs

```diff
@@ -25,7 +25,7 @@ public static async Task Main(string[] args)
 		{
 			await Parser.Default.ParseArguments<Options>(args)
 				.MapResult(
-				(Options options) => DynatraceExporterExample.RunAsync(options), errs => Task.FromResult(0));
+				(Options options) => DynatraceExporterExample.RunAsync(options), _ => Task.FromResult(1));
```

Comment:
Is the `1` the return code?

Reply:
This was what @pichlermarc suggested. When in the error callback, we return `1` to signal error

Reply:
Yes, but I was mistaken. I tried it and this does not return the correct error code, it is still `0`. Sorry about that.

Reply:
So even if we fail we return a 0 error code?

Reply:
Yes. This would return the correct one:

```cs
_ =>
{
  // actually exits
  Environment.Exit(1);
  // So that it still returns a task.
  return Task.FromResult(0);
}
```

I do not know if it is "nice" though.

Reply:
This would probably be better:
```cs
var result = await Parser.Default.ParseArguments<Options>(args)
  .MapResult(
  (Options options) => DynatraceExporterExample.RunAsync(options), _ => Task.FromResult(1));
Environment.Exit(result);
```

Reply:
done.

Reply:
Yeah, i think the second approach is better. The only question is if we want to crash at all, or just go with the default values and log that the arg could not be parsed. Either way is fine by me, since this is just the example app.